modules/virt-access-configuration-considerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
```

Comment:
@phoracek Can you review "Access configuration considerations"?

CC @dominikholler